pkg/server/auth/server.go

```diff
@@ -110,9 +120,29 @@ func NewOIDCConfigFromSecret(secret corev1.Secret) OIDCConfig {
 
 	cfg.TokenDuration = tokenDuration
 
+	scopes := splitAndTrim(string(secret.Data["customScopes"]))
+	if len(scopes) == 0 {
```

Comment:
I think I may have been doing the odd `if slice == nil` in other places, will keep the distinction in mind..

Reply:
In this case, `result := []string{}` will return a slice of zero length.

`var result []string` would mean that result _could_ be nil.

Taking the length of a nil slice yields `0` any which way.